audit/projects/k8s-infra-e2e-boskos-scale-01/buckets/artifacts.k8s-infra-e2e-boskos-scale-01.appspot.com/bucketpolicyonly.txt

```diff
@@ -0,0 +1,3 @@
+Bucket Policy Only setting for gs://artifacts.k8s-infra-e2e-boskos-scale-01.appspot.com:
```

Comment:
If we scripted this, the script should init these for every project, no?

Reply:
Probably a good idea

After some of the digging I did into GCR/GCB permissions in https://github.com/kubernetes/test-infra/issues/20884#issuecomment-783712871 I'd like to bucket this under a "we need to revisit GCR/GCB permissions" issue

I wasn't around for the initial set of scripts that set us down the path of bucket pre-creation, but I gather we want to be more conservative with `storage.buckets.(create|delete|update)` permissions than the default roles provided by these services?